php/conf.d/custom.ini

```diff
@@ -13,15 +13,16 @@ display_errors = Off
 error_log = /dev/stderr
 expose_php = Off
 log_errors = On
-max_execution_time = 18000
+max_execution_time = 3600
 memory_limit = 256M
 realpath_cache_size = 1M
 realpath_cache_ttl = 3600
 sendmail_path = /usr/sbin/ssmtp -t
 short_open_tag = Off
 SMTP = maildev
 smtp_port = 25
-upload_max_filesize = 120M
+upload_max_filesize = 100M
+post_max_size = 100M
 
 [date]
 date.timezone = UTC
```

Comment:
I don't see the Xdebug configuration that we have in the previous `custom.ini`. It has been moved to an other file ?

Reply:
It's corrected